tests/integration-tests/cfn_stacks_factory.py

```diff
@@ -307,6 +307,21 @@ def delete_all_stacks(self):
                     )
                 )
 
+    def delete_stacks_retain_ad(self):
+        """Destroy all created stacks."""
```

Comment:
This function destroyes all stack but VPC, and AD stacks. The description should be aligned with this.